templates/webauthn.twig

```diff
@@ -24,21 +22,20 @@
             </ul>
         </div>
     {% endif %}
-    {% if regForm|length > 0 %}
+    {% if regURL|length > 0 %}
         <form id='regform' method='POST' action='{{ regURL|raw }}'>
             <input type='hidden' id='resp' name='response_id' value='0'/>
             <input type='hidden' id='data' name='attestation_client_data_json' value='nix'/>
             <input type='hidden' id='attobj' name='attestation_object' value='mehrnix'/>
             <input type='hidden' id='type' name='type' value='something'/>
             <input type='hidden' id='operation' name='operation' value='REG'/>
-            <button type='button' onClick="{{ regForm|raw }}" onsubmit='false' >{{ '{webauthn:webauthn:newTokenButton}'|trans }}</button>
+            <button type='button' id='regformSubmit'>{{ '{webauthn:webauthn:newTokenButton}'|trans }}</button>
             {{ '{webauthn:webauthn:newTokenName}'|trans }} <input 
                 type='text' 
                 id='tokenname' 
                 name='tokenname' 
                 size='40' 
-                value='{{ '{webauthn:webauthn:newTokenDefaultName}'|trans }} {{ "now"|date('Y-m-d', timezone="Europe/Luxembourg") }}' onkeydown="if (event.keyCode == 13)
-                            return false;"/>
+                value='{{ '{webauthn:webauthn:newTokenDefaultName}'|trans }} {{ "now"|date('Y-m-d', timezone="Europe/Luxembourg") }}' />
```

Comment:
Same here.

Also, I don't think we should be fixing the timezone here. I don't know why it's like that, but it definitely shouldn't be the case.

Reply:
Sorry for the timezone, it's a leftover from my initial local deployment. This should of course be the globally configured timezone of simpleSAMLphp as per main config file.